src/tracker/timelineSelectors.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import { createSelector } from 'reselect'
+import type { InputSelector } from 'reselect'
+import moment from 'moment'
+import { reduce } from 'lodash'
+
+import eventsSelector from './eventsSelector'
+import displayedDateSelector from './displayedDateSelector'
+
+import type { State } from './../store/state'
+
+import { formatTime } from './utils'
+
+export const timelineEventsSelector: InputSelector<
+  State,
+  any,
+  Event[]
+> = createSelector(
+  [eventsSelector, displayedDateSelector],
+  (events, displayedDate) => {
+    const filteredEvents = events.filter(event =>
+      moment(event.start).isSame(displayedDate, 'day')
+    )
+    return filteredEvents.length
+      ? filteredEvents
+      : [
+          {
+            ...events[events.length - 1],
+            start: moment(displayedDate).startOf('day')
+          }
+        ]
+  }
+)
+
+export const hoursLoggedPerDaySelector: InputSelector<
```

Comment:
this doesn't return hours logged per day, this only returns that on one day - the selected one. Since we are already in file named `timelineSelectors`, I think it's enough to just name like this:
```suggestion
export const hoursLoggedSelector: InputSelector<
```